data/channels.csv

```diff
@@ -12486,8 +12489,8 @@ KanalS.tr,Kanal S,,,,TR,,Samsun,c/TR,tur,news,FALSE,,,,https://www.kanals.com.tr
 KanalSim.cy,Kanal Sim,,,,CY,,,c/CY,tur,,FALSE,,,,http://www.kanalsim.com/,https://i.imgur.com/UHYJURw.png
 KanalTuran.fr,Kanal Turan,,,,FR,,Strasbourg,c/FR;c/AZ,aze,news,FALSE,,,,https://www.kanalturan.com/,https://i.imgur.com/9AWwu2o.png
 KanalUrfa.tr,Kanal Urfa,,,,TR,,,c/TR,tur,,FALSE,,,,https://kanalurfa.com/,https://i.imgur.com/EvgQmoT.png
-KanalV.tr,Kanal V,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/7RNjTno.png
-KanalZ.tr,Kanal Z,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/Kr8960d.png
+KanalV.tr,Kanal V,,,,TR,,Antalya,c/TR,tur,,FALSE,,,,https://www.kanalv.com.tr/,https://i.imgur.com/o26IDy4.png,https://i.imgur.com/7RNjTno.png

```

Comment:
why 2 imgur images?

Reply:
I'm not sure if it would be better to keep the logo according to the standard it complies with. What do you think? @Dum4G

Reply:
As long as it is up to 512x512 png image that you feel is readable to human eye I'm okay with it. But there should be only one of them, current implementation will break the database

Reply:
I will fix the appropriate one when it becomes available.